core-build/gulp-core-build-typescript/src/ApiExtractorTask.ts

```diff
@@ -6,7 +6,7 @@ import {
   JsonFile,
   FileSystem
 } from '@microsoft/node-core-library';
-import { ApiExtractorRunner } from '@microsoft/rush-stack-compiler';
+import { ApiExtractorRunner as TApiExtractorRunner } from '@microsoft/rush-stack-compiler';
```

Comment:
>TApiExtractorRunner [](start = 31, length = 19)

Why is this renamed?

Reply:
I want to specifically designate that this should only be used as a type.

---
In reply to: [234095868](https://github.com/Microsoft/web-build-tools/pull/947#discussion_r234095868) [](ancestors = 234095868)